static/js/narrow.js

```diff
@@ -833,6 +835,7 @@ function handle_post_narrow_deactivate_processes() {
     }
 
     top_left_corner.handle_narrow_deactivated();
+    pm_list.handle_narrow_deactivated();
```

Comment:
The change to extract a `handle_narrow_deactivated` function (moving the `pm_list` related code currently in `top_left_corner.handle_narrow_deactivated`) without any other changes would be a good prep commit that we can split out and integrate before the rest of this PR to reduce its complexity.  There may be a few others as well.

Reply:
Got it, Sure -- this can come out earlier as a prep commit to reduce this PR's complexity.